pyproject.toml

```diff
@@ -36,7 +36,9 @@ dependencies = [
 dynamic = ["version"]
 [project.urls]
 "Homepage" = "https://github.com/lsst/daf_butler"
+
 [project.optional-dependencies]
+postgres = ["psycopg2"]
```

Comment:
@r-owen I think this is what you want. You should be able to do `pip install lsst-daf-butler[postgres]` and get the optional dependency installed.

Reply:
When I try `lsst-daf-butler[postgres]` I get:
```
no matches found: lsst-daf-butler[postgres]
```
If I remove the `[postgres]` and just run `pip install lsst-daf-butler` then it starts to install things. Any idea what's going on with that?

Reply:
Found it. The package name needs to be quoted: `pip install "lsst-daf-butler[postgres]"`

Reply:
Yes. Because this MR is not yet merged and uploaded to PyPi.

Reply:
So how can I test it locally?

Reply:
Best to approve the pull request and I trigger uploading to PyPi

Reply:
if you clone it and check out the branch you can do `pip install .[postgres]`

Reply:
Thank you. I will do that.

Reply:
Turns out that also needed quotes, e.g. `pip install ".[postgres]"` but then it worked.